### PR TITLE
Introduce getters for LookupFailedExcpetion properties name and type

### PR DESCRIPTION
**Pull request by nresare**
Oops. Forgot this.

### CODE REVIEW

File: src/test/java/org/xbill/DNS/lookup/LookupFailedExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package org.xbill.DNS.lookup;
```
Comment (ibauersachs) [inline src/test/java/org/xbill/DNS/lookup/LookupFailedExceptionTest.java:1]: Please add the SPDX license header

**Review by ibauersachs — APPROVED** [review]
I missed that the header is missing on some of the new files of #162 as well:
- LookupSessionTest.java
- NoSuchRRSetException.java
- NoSuchDomainException.java 

We should probably add the header to all existing files as well, regardless of what is already there and enforce it using a checkstyle rule.

File: src/test/java/org/xbill/DNS/lookup/LookupFailedExceptionTest.java (hunk 1 above)
Comment (nresare) [reply src/test/java/org/xbill/DNS/lookup/LookupFailedExceptionTest.java:1]: Sure. Added.

**Comment by nresare** [comment]
That sounds like a plan :)

**Pull request merged**